sdk/resourcemanager/appservice/armappservice/README.md

```diff
@@ -50,6 +50,12 @@ options = arm.ClientOptions{
 client := armappservice.NewWebAppsClient(<subscription ID>, cred, &options)
 ```
 
+## More sample code
+
+- [Static Site](https://aka.ms/azsdk/go/mgmt/samples?path=appservice/staticsite)
+- [App Service Plan](https://aka.ms/azsdk/go/mgmt/samples?path=appservice/web_plans)
```

Comment:
Seems this link does not work.

Reply:
This change is pending on a change in the sample repo. In the sample repo, the samples for appservice was previously in the `web` directory. We are renaming it. When it is done, these links should be available again